src/command_line_parser.cc

```diff
@@ -380,6 +382,11 @@ std::vector<Option> TritonParser::recognized_options_
        "Allow the server to listen for HTTP requests."},
       {OPTION_HTTP_PORT, "http-port", Option::ArgInt,
        "The port for the server to listen on for HTTP requests."},
+      {OPTION_HTTP_HEADER_FORWARD_PREFIX, "http-header-forward-prefix",
```

Comment:
same as grpc flag comment, revert back to pattern/regex arg name and description